src/constants.zig

```diff
@@ -436,7 +436,7 @@ comptime {
     assert(journal_iops_read_max > 0);
     assert(journal_iops_write_max > 0);
     assert(client_replies_iops_read_max > 0);
-    assert(client_replies_iops_write_max > 0);
+    // Require at least two: one repair, and one commit.
```

Comment:
Deleted wrong line?

Reply:
:facepalm: I deleted the line but it should have been an edit to restore the original condition (and remove the comment).